File: WebApp/Scripts/app/model/TC_PURCH_DL.js
Ext.define('WEBAPP.model.TC_PURCH_DL', {
    extend: 'Ext.data.Model',
    fields: [
        { name: 'PUR_NO', type: 'string' },
        { name: 'MMCODE', type: 'string' },
        { name: 'AGEN_NAMEC', type: 'string' },
        { name: 'MMNAME_C', type: 'string' },
        { name: 'PUR_QTY', type: 'string' },
        { name: 'PUR_UNIT', type: 'string' },
        { name: 'IN_PURPRICE', type: 'string' },
        { name: 'PUR_AMOUNT', type: 'string' },


        { name: 'CREATE_TIME', type: 'string' },
        { name: 'CREATE_USER', type: 'string' },
        { name: 'UPDATE_TIME', type: 'string' },
        { name: 'UPDATE_USER', type: 'string' },
        { name: 'UPDATE_IP', type: 'string' },

        { name: 'BASE_UNIT', type: 'string' },
        { name: 'M6AVG_USEQTY', type: 'string' },
        { name: 'INV_DAY', type: 'string' },
        { name: 'PURUN_MULTI', type: 'string' },
        { name: 'BASEUN_MULTI', type: 'string' },
        { name: 'PUR_DAY', type: 'string' },
        { name: 'ORI_AGEN_NAMEC', type: 'string' },
        { name: 'ORI_PUR_QTY', type: 'string' },
        { name: 'ORI_PUR_UNIT', type: 'string' },
        { name: 'ORI_IN_PURPRICE', type: 'string' },
        { name: 'ORI_PUR_AMOUNT', type: 'string' },


    ]
});

File: WebApp/Models/TC/TC_PURCH_DL.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace WebApp.Models
{
    public class TC_PURCH_DL
    {
        public string PUR_NO { get; set; }
        public string MMCODE { get; set; }
        public string AGEN_NAMEC { get; set; }
        public string MMNAME_C { get; set; }
        public string PUR_QTY { get; set; }
        public string PUR_UNIT { get; set; }
        public string IN_PURPRICE { get; set; }
        public string PUR_AMOUNT { get; set; }


        public string CREATE_TIME { get; set; }
        public string CREATE_USER { get; set; }
        public string UPDATE_TIME { get; set; }
        public string UPDATE_USER { get; set; }
        public string UPDATE_IP { get; set; }



        public string BASE_UNIT { get; set; }       // 計量單位
        public string M6AVG_USEQTY { get; set; }    // 前6個月平均消耗量
        public string INV_DAY { get; set; }         // 庫存天數
        public string ORI_AGEN_NAMEC { get; set; }         // 原始廠商
    }
}